tests/std/tests/P0355R7_calendars_and_time_zones_formatting/test.cpp

```diff
@@ -36,20 +36,6 @@ struct choose_literal<wchar_t> {
 
 #define TYPED_LITERAL(CharT, Literal) (choose_literal<CharT>::choose(Literal, L##Literal))
 

```

Comment:
I am concerned about the cases where we encounter the closing brace or end of the string - I would like to see `err_expected` coverage below for each scenario.